### PR TITLE
Imagery layer alpha control

### PR DESCRIPTION
**Pull request by lilleyse**
Added `alpha` property to `CesiumImageryPrim` as a quick and easy way to control the translucency of an imagery layer.

Technically this could also be achieved with tileset materials (like the example [here](https://github.com/CesiumGS/cesium-omniverse/pull/406#issuecomment-1771561788)) but it would be slower and would require a tileset reload (currently).

653294bcde1da120674075

For testing: [multiple.usda.zip](https://github.com/CesiumGS/cesium-omniverse/files/13055753/multiple.usda.zip)

### CODE REVIEW

File: exts/cesium.usd.plugins/schemas/cesium_schemas.usda [hunk 1]
```diff
@@ -371,6 +371,14 @@ class CesiumImageryPrim "CesiumImageryPrim" (
         displayName = "Show Credits On Screen"
         doc = "Whether or not to show this imagery layer's credits on screen."
     )
+
+    float cesium:alpha = 1.0 (
+        customData = {
+            string apiName = "alpha"
+        }
+        displayName = "Alpha"
+        doc = "The alpha blending value, from 0.0 to 1.0."
```
Comment (weegeekps) [inline exts/cesium.usd.plugins/schemas/cesium_schemas.usda:380]: NIT: We should probably mention in the doc that `1.0` means fully opaque.

**Review by weegeekps — APPROVED** [review]
One nitpick but otherwise looks good to me.

File: src/core/src/FabricPrepareRenderResources.cpp [hunk 2]
```diff
@@ -467,7 +467,7 @@ void FabricPrepareRenderResources::attachRasterInMainThread(
 
             const auto texcoordIndex = mesh.imageryTexcoordIndexMapping.at(gltfSetIndex);
             const auto& textureAssetPath = texture->getAssetPathToken();
-            material->setImageryLayer(textureAssetPath, textureInfo, texcoordIndex, imageryLayerIndex.value());
+            material->setImageryLayer(textureAssetPath, textureInfo, texcoordIndex, imageryLayerIndex.value(), 1.0f);
```
[the remark this answers is not in the document]
Comment (weegeekps) [reply src/core/src/FabricPrepareRenderResources.cpp]: Oh, I thought that was intentional. Ok, I'll review again when you've pushed a fix.

[the remark this answers is not in the document]
Comment (lilleyse) [reply src/core/src/FabricPrepareRenderResources.cpp]: https://github.com/CesiumGS/cesium-omniverse/pull/498/commits/d064e5c5b936b8fc163e2ff3aa1581e78ee4b557 - ready for review

**Pull request merged**